### PR TITLE
APIGOV-11960 OAuth Client Authentication Documentation

### PR DESCRIPTION
**Pull request by SteveSonoa**

### CODE REVIEW

**Comment by SteveSonoa** [comment]
@robertengaxway @markbice @alexearnshaw @andreamussap

File: content/en/docs/central/feauth_oauth [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+### Creating An AMPLIFY Central Application Profile
+
+After you have created an application within your 3rd party OAuth server, you will need to provide AMPLIFY Central with some of the information in order to properly use your secured proxy.
+
+- Visit the Apps section of AMPLIFY Central. You can create a new app or use an existing one.
+- Click the "Identity Profiles" tab.
+- Click the "+ OAuth Profile" button on the right side of the screen.
+
+This will open a new dialog box for your OAuth information. Choose a helpful name and description. The other fields (Client ID, Issuer, and Metadata Path) should be provided by your 3rd party OAuth server.
```
Comment (georgiana-caras) [inline content/en/docs/central/feauth_oauth]: Are we currently supporting only Okta? Can we document or have a short video on how Oauth works with Okta?

File: content/en/docs/central/feauth_oauth [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: OAuth Proxy Policy
+linkTitle: OAuth Proxy Policy
+weight: 2
+date: 2019-07-30
+description: Learn how to setup an OAuth Front End Security policy on a proxy in AMPLIFY Central.
```
Comment (georgiana-caras) [inline content/en/docs/central/feauth_oauth]: To be consistent, use API proxy everywhere.

File: content/en/docs/central/feauth_oauth (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/central/feauth_oauth]: We have only tested Okta but we should make references to "third party Identity Provider (IdP) support"   or "OAuth server"

**Comment by andreamussap** [comment]
@SteveSonoa I'll perform the editorial review after the stakeholders perform the technical review, and you close all opened comments.  Thanks, Andrea.

File: content/en/docs/central/feauth_oauth (hunk 1 above)
Comment (markbice) [reply content/en/docs/central/feauth_oauth]: There will need to be some part of a video (before we are done) that _does_ show the Okta config needed to make this fly. Not how to install Okta, but exactly how the config we use in Central is related.

**Comment by markbice** [comment]
Does this branch only include the CLI changes, or does is also include the UI/OAuth updates for Proxy/Policies tab and App/Profiles tab?

File: content/en/docs/central/feauth_oauth (hunk 1 above)
Comment (SteveSonoa) [reply content/en/docs/central/feauth_oauth]: I'd value your input on how to accomplish that, and what the end result should look like. I was specifically asked not to reference Okta directly in the docs, but "third party providers" instead. I'm happy to make a video that meets all needs, but I need direction on what that looks like and where it belongs.

**Comment by SteveSonoa** [comment]
> Does this branch only include the CLI changes, or does is also include the UI/OAuth updates for Proxy/Policies tab and App/Profiles tab?

This branch did not involve CLI update. I only worked on the web UI.

File: content/en/docs/central/feauth_oauth (hunk 2 above)
Comment (SteveSonoa) [reply content/en/docs/central/feauth_oauth]: Will do; thank you.

**Comment by SteveSonoa** [comment]
I added a note that briefly indicates Okta is our only OAuth provider at this time. This part of the documentation should be updated when additional providers are included.

File: content/en/docs/central/feauth_oauth (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/central/feauth_oauth]: @markbice  I asked Steve to create a video for internal use for now.   A suggestion here might be to provide a Github location for the latest instructions on how to configure Okta/Ping/V7/etc in a similar way to how we handled the Service Mesh installation instructions.

**Comment by SteveSonoa** [comment]
I added an additional screenshot of the Identity Profile at @robertengaxway 's request.

**Comment by andreamussap** [comment]
@SteveSonoa and others -  I've performed an editorial review but it's not over. On this revision I:

- Renamed file feauth_oauth (added .md). It wasn't showing in the preview because of this.
- Added sections Before you start and Objective.
- Reviewed/ modified parts of the text

I need you to technically review what I added/changed and:

- Remove screenshot from 'Create an application profile' because the image is not relevant
- Consider if this title suits best - 'Secure an API with OAuth'??  (e.g., [Secure an API with JWT](https://axway-open-docs.netlify.com/docs/central/secure_api_jwt/)
- Section 'Set up OAuth security on an API proxy': it would be more helpful if you added a one animated gif than three static images.

This is the direct link to the preview - https://deploy-preview-199--axway-open-docs.netlify.com/docs/central/feauth_oauth/

**Comment by andreamussap** [comment]
@SteveSonoa @markbice what's missing to finish this task?  Please review the content and verify that's good so we can merge it to production.

Steve you must '**Resolve**' all comments here otherwise I can't approve/merge the PR. 

Direct link to the preview - https://deploy-preview-199--axway-open-docs.netlify.com/docs/central/feauth_oauth/

**Comment by SteveSonoa** [comment]
@andreamussap Thank you for the direction on resolving comments; that has been completed. After talking with @robertengaxway it looks like the customer-facing videos (which have been completed) will be uploaded and updated at a later time (in a future story/task), as we're not sure where they should currently be located. I believe all other comments have been resolved. Just looking to @markbice for final approval.

**Comment by SteveSonoa** [comment]
@andreamussap Looks like this review is ready. Please let me know if there's anything else you need me to do. Thank you!

**Comment by andreamussap** [comment]
### Thank you
We really appreciate your contribution and thank you for helping us to improve this documentation!
 
### Where to find the updated documentation
An Axway technical writer has reviewed and merged your PR, which is now available on the following sites:
Netlify (staging): https://axway-open-docs.netlify.com/docs/
Zoomin (production): https://docs.axway.com/bundle/axway-open-docs/
 
### Send us your feedback
If you have a moment, please complete the following 2 minute survey to give us your feedback on this process and help us make it even better:
https://www.surveymonkey.com/r/PJ5HQFD

**Pull request merged**